EntertainmentVenue.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class EntertainmentVenue {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input the age of costumer");
+        int age = sc.nextInt();
+        if (age < 16) {
+            System.out.println("The entry of the costumer is banned");
+            System.exit(0);
+        } else {
```

Comment:
We don't need `else` here. We write `if-else` to normalize some aspects that help the program to work correctly, but not to put all the code inside them.